app/models/item_order.rb

```diff
@@ -4,6 +4,8 @@ class ItemOrder <ApplicationRecord
   belongs_to :item
   belongs_to :order
 
+  enum status: %w(unfulfilled fulfilled)
```

Comment:
glad to see you went with an enum, I think it will give us more flexibility than a boolean